Makefile

```diff
@@ -4,11 +4,11 @@ TAG = v0.20.0-alpha8
 PREFIX = gcr.io/google_containers
 FLAGS = 
 
-SUPPORTED_KUBE_VERSIONS = "1.0.6"
+SUPPORTED_KUBE_VERSIONS = ""
```

Comment:
Really? IN its current state, it should run the e2e tests against an existing cluster.